internal/monitor/yahoo/unary/helpers-currency.go

```diff
@@ -0,0 +1,173 @@
+package unary
+
+// minorCurrency represents the scaling ('minor unit') for a minor currency.
+// Convert major to minor by multiplying by 10^n and minor to major by dividing by 10^n.
+type minorCurrency struct {
+	MajorCurrencyCode string
+	MinorCurrencyCode string
+	MinorUnit         float64
+}
+
+// This map is derived from https://www.six-group.com/dam/download/financial-information/data-center/iso-currrency/lists/list-one.xls
+var _minorCurrencyCodeByMajorCurrencyCode = map[string]minorCurrency{
```

Comment:
Could this list be limited to only trading currencies for major exchanges? I expect most of these currencies do not have trading pairs on any exchanges so the list could be slimed down. This like should be a good start and based on reported issues, others can be added.

```
USD, JPY, CNY, HKD, GBP, EUR, CAD, INR, AUD, CHF, KRW, TWD, ZAR
```